curriculum/challenges/english/25-front-end-development/lecture-understanding-asynchronous-programming/67340798c2c1776709d8a5fe.md

```diff
@@ -46,35 +46,35 @@ In short, use `async` for scripts where the order of execution doesn't matter, a
 
 ## --text--
 
-What happens when a script tag has both `async` and `defer` attributes?
+What is the advantage of using the `defer` attribute in a `<script>` tag?
 
 ## --answers--
 
-The script is downloaded asynchronously and executed immediately after download.
+It helps differentiate the script from `async` scripts.
 
 ### --feedback--
 
-Consider which attribute takes precedence when both are present.
+Remember the purpose and differences of the `defer` and `async` attributes.
 
 ---
 
-The script is downloaded asynchronously and executed after HTML parsing is complete.
+It reduces memory usage during page load.
 
 ### --feedback--
 
-Consider which attribute takes precedence when both are present.
+The `defer` attribute has nothing to do with memory management.
 
 ---
 
-The `defer` attribute is ignored, and the script behaves as if only `async` was present.
+It improves the overall page loading performance.
```

Comment:
```suggestion
It can improve the overall page loading performance.
```
Depending where the `<script>` tag without `defer` is located, it might not affect loading performance, hence suggested wording to present improved performance as possibility.